src/sql-parser/src/parser.rs

```diff
@@ -4178,10 +4179,15 @@ impl<'a> Parser<'a> {
                 Some(NOCREATEDB) => options.push(RoleAttribute::NoCreateDB),
                 Some(CREATEROLE) => options.push(RoleAttribute::CreateRole),
                 Some(NOCREATEROLE) => options.push(RoleAttribute::NoCreateRole),
+                Some(PASSWORD) => {
+                    //TODO(dov): allow unsetting password with null
```

Comment:
is this a blocker?

Reply:
IMO not for merging this PR, but we'd probably want to add this before releasing to self-hosted customers? Not having a way to unset a password feels bad!

Reply:
Does setting a password make the role a loginrole? If so we should definitely support unset.